contracts/auction.sol

```diff
@@ -15,6 +15,8 @@ contract DutchAuction {
      * token_multiplier set from token's number of decimals (i.e. 10 ** decimals)
      */
 
+    uint constant public waiting_period = 7 days;
```

Comment:
Can you rename to `token_claim_waiting_period`?

Reply:
will do.